cpan_deps_graph.pl

```diff
@@ -96,16 +105,21 @@
   }
 };
 
-helper get_dist_deps => sub ($c, $dist, $phases, $relationships, $perl_version) {
+helper get_dist_deps => sub ($c, $dist, $phases, $relationships, $perl_version, $dist_version = undef) {
   $perl_version = $perl_version->numify;
   my $redis = $c->redis->db;
   my %all_deps;
   foreach my $phase (@$phases) {
     foreach my $relationship (@$relationships) {
-      my $key = "cpandeps:$dist:$phase:$relationship";
-      my $deps_json = $redis->get($key) // next;
       my $deps;
-      try { $deps = from_json $deps_json } catch { next }
+      if ($dist_version) {
```

Comment:
This needs to be 0-safe - at the very least, check length $dist_version if not defined

Reply:
Assuming you mean "if defined" - agreed, will execute.